src/modules/Lan.rb

```diff
@@ -186,16 +184,12 @@ def isAnyInterfaceDown
           )
         ) do |devname|
           mac = Ops.get_string(
-            Convert.convert(
-              SCR.Execute(
-                path(".target.bash_output"),
-                Builtins.sformat(
-                  "cat /sys/class/net/%1/address|tr -d '\n'",
-                  devname
-                )
-              ),
-              from: "any",
-              to:   "map <string, any>"
+            SCR.Execute(
```

Comment:
All this can be replaced with 
```rb
mac = File.read("/sys/class/net/#{devname}/address").chomp
```
YCP forever!

Reply:
I expect we do not need to care that File is locally and SCR::Execute on target system as sys should be mount binded, right? BTW if we need to have chroot aware file, then this one can be used: https://github.com/yast/yast-yast2/blob/30a9253009a02a8fb0366c372fe82cdb746dfd40/library/system/src/lib/yast2/target_file.rb